pymagicc/__init__.py

```diff
@@ -301,8 +296,9 @@ def run(scenario, return_config=False, **kwargs):
         if 'endyear' in kwargs:
             year_cfg['endyear'] = kwargs.pop('endyear')
             magicc.set_years(**year_cfg)
-
-        kwargs.setdefault(get_param('emission_scenario_key'), "SCENARIO.SCEN")
+        emis_key = "file_emissionscenario" if magicc.version == 6 \
```

Comment:
Could this check be done in the class, if run with `MAGICC6` there should be no need to check for the version again, should it? Or it could be `MAGICC` and then depend on the MAGICC version set -- but probably `run` should always use one version by default (if Pymagicc shipped with 6 and 7)

Reply:
True in this case the version check is not needed at all since it will always be using MAGICC6.

BTW: The MAGICC base class shouldn't be used - it is an abstract class since it doesn't define `get_executable`.